plugins/inputs/opentelemetry_listener/grpc_services.go

```diff
@@ -0,0 +1,72 @@
+package opentelemetry_listener
+
+import (
+	"context"
+
+	"github.com/influxdata/influxdb-observability/otel2influx"
+	otlpcollectorlogs "github.com/influxdata/influxdb-observability/otlp/collector/logs/v1"
+	otlpcollectormetrics "github.com/influxdata/influxdb-observability/otlp/collector/metrics/v1"
+	otlpcollectortrace "github.com/influxdata/influxdb-observability/otlp/collector/trace/v1"
```

Comment:
You might want to skip `otlp` prefix.

Reply:
The `otlp` prefix is consistent with the usual aliasing of those packages. For example:
https://github.com/open-telemetry/opentelemetry-collector/blob/1be517f55558f6983a55ae6145e7697a01769e48/exporter/otlpexporter/otlp.go#L34-L36